v4/export/writer.go

```diff
@@ -126,25 +139,38 @@ type csvOption struct {
 
 func newOutputFileNamer(ir TableDataIR) *outputFileNamer {
 	return &outputFileNamer{
-		chunkIndex: ir.ChunkIndex(),
-		dbName:     ir.DatabaseName(),
-		tableName:  ir.TableName(),
+		Index: ir.ChunkIndex(),
+		DB:    ir.DatabaseName(),
+		Table: ir.TableName(),
 	}
 }
 
-func (namer *outputFileNamer) NextName() string {
-	defer func() { namer.chunkIndex++ }()
-	if namer.dbName == "" || namer.tableName == "" {
-		return fmt.Sprintf("result.%d", namer.chunkIndex)
+func (namer *outputFileNamer) NextName(tmpl *template.Template) (string, error) {
+	defer func() { namer.Index++ }()
+	if namer.DB == "" || namer.Table == "" {
+		return fmt.Sprintf("result.%d", namer.Index), nil
```

Comment:
🤔

Reply:
How about the fix in https://github.com/pingcap/dumpling/pull/120/commits/12d0e06fbdc477d70077965249ddb6af82a66147?